test/unit/specs/components/governance/LiProposal.spec.js

```diff
@@ -1,80 +1,134 @@
-import setup from "../../../helpers/vuex-setup"
-import lcdClientMock from "renderer/connectors/lcdClientMock.js"
+import { shallowMount, createLocalVue } from "@vue/test-utils"
 import LiProposal from "renderer/components/governance/LiProposal"
 
-const { proposals, tallies } = lcdClientMock.state
-const proposal = proposals[`2`]
-
-const $store = {
-  commit: jest.fn(),
-  dispatch: jest.fn(),
-  getters: {
-    proposals: { proposals, tallies }
+const proposals = {
+  1: {
+    proposal_status: `Passed`,
+    proposal_id: `1`,
+    title: `Proposal Title`,
+    description: `Proposal description`,
+  },
+  2: {
+    proposal_status: `VotingPeriod`,
+    proposal_id: `2`,
+    title: `VotingPeriod proposal`,
+    description: `custom text proposal description`,
+  },
+  tallies: {
+    1: {
+      yes: `500`,
+      no: `25`,
+      no_with_veto: `10`,
+      abstain: `56`
+    },
+    2: {
+      yes: `0`,
+      no: `0`,
+      no_with_veto: `0`,
+      abstain: `0`
+    }
   }
 }
 
+const proposal = proposals[`2`]
+
 describe(`LiProposal`, () => {
+  const localVue = createLocalVue()
+  localVue.directive(`tooltip`, () => { })
+
   let wrapper
-  const { mount } = setup()
 
   beforeEach(() => {
-    const instance = mount(LiProposal, {
-      doBefore: ({ store }) => {
-        store.commit(`setConnected`, true)
-        store.commit(`setProposal`, proposal)
-        store.commit(`setProposalTally`, {
-          proposal_id: `2`,
-          final_tally_result: tallies[`2`]
-        })
+    const $store = {
+      commit: jest.fn(),
+      dispatch: jest.fn(),
+      getters: {
+        proposals
+      }
+    }
+
+    wrapper = shallowMount(LiProposal, {
+      localVue,
+      mocks: {
+        $store
       },
       propsData: { proposal },
-      $store
+      stubs: [`router-link`]
     })
-    wrapper = instance.wrapper
   })
 
   it(`has the expected html structure`, () => {
     expect(wrapper.vm.$el).toMatchSnapshot()
   })
 
   it(`should return status info for passed proposals`, () => {
-    proposal.proposal_status = `Passed`
-    wrapper.setProps({ proposal: JSON.parse(JSON.stringify(proposal)) })
+    wrapper.setProps({
```

Comment:
We could do something like: 
```
proposal = {stuff}
...
{
  proposal: {
    ...proposal,
   proposal_status: `the status I want to test`
  }
}
```

Reply:
same comment about cloning

Reply:
i don't understand what you guys are suggesting.

Reply:
Karoly is proposing that you change the proposal instantiation. His way the difference between the proposals can be spotted quicker. It also limits the code duplication. Ignore my comment.

Reply:
done.